src/main/java/nl/cherement/jak/service/CardService.java

```diff
@@ -22,6 +29,23 @@ public CardService(CardRepository repository) {
         return this.repository.findByColumnId(id);
     }
 
+    @Override
+    public CardEntity save(Authentication user, CardEntity o) {
```

Comment:
Waarom halen we de cardentity opnieuw op?

Reply:
omdat je moet checken of de columnId veranderd is. CardEntity o is de nieuwe aagepaste. Dus je moet eerste de oude ophalen om te kunnen zien wat de columnId nu is